.travis.yml

```diff
@@ -6,7 +6,7 @@ cache:
 notifications:
   email: false
 node_js:
-  - '4'
+  - stable
```

Comment:
I have to admit, I hadn't looked it up. But I'm sure there's sth like `latest` just don't remember the exact dist-channel. Will look it up when I find the time (but I think, this is nothing that should hold this PR back).